src/Worker/Notifier.php

```diff
@@ -441,7 +441,7 @@ function_exists('imap_open')
 						continue;
 					}
 
-					if (in_array($rr['network'], [Protocol::DFRN, Protocol::DIASPORA]) && ($rr['protocol'] == Protocol::ACTIVITYPUB) &&
+					if (!empty($rr['network']) && in_array($rr['network'], [Protocol::DFRN, Protocol::DIASPORA]) && ($rr['protocol'] == Protocol::ACTIVITYPUB) &&
```

Comment:
Not the network, but the protocol is missing sometimes. The PR https://github.com/friendica/friendica/pull/7205 solves that.

Reply:
Good catch.